lib/Parser/Parse.cpp

```diff
@@ -2435,12 +2435,37 @@ bool Parser::IsImportOrExportStatementValidHere()
         && this->m_tryCatchOrFinallyDepth == 0;
 }
 
+template<bool buildAST> ParseNodePtr Parser::ParseImportCall()
+{
+    m_pscan->Scan();
+    ParseNodePtr specifier = ParseExpr<buildAST>(koplCma, nullptr, /* fAllowIn */FALSE, /* fAllowEllipsis */FALSE);
```

Comment:
Yes, it stops at ',' and doesn't accept '...'

---
In reply to: [115330698](https://github.com/Microsoft/ChakraCore/pull/2913#discussion_r115330698) [](ancestors = 115330698)